src/pkg/projects/dir.targets

```diff
@@ -120,4 +122,62 @@
     </MSBuild>
   </Target>
 
+  <!--
+    Creates the platform manifest and props file. Set properties "IncludePlatformManifestFile" and
+    "IncludeFileVersionPropsFile" to true to include them, respectively.
+  -->
+  <Target Name="GenerateFileVersionProps">
+    <PropertyGroup>
+      <PropsFile>$(IntermediateOutputPath)$(MSBuildProjectName).props</PropsFile>
+      <PlatformManifestFile>$(IntermediateOutputPath)$(MSBuildProjectName).PlatformManifest.txt</PlatformManifestFile>
+
+      <!-- During an official build when we can guarantee that all RID-specific dependencies have been built,
+          restore all of those dependencies and gather the prospective content of the RID-specific Core.App
+          packages.  This is needed so that we have a complete platform manifest in the shipping version of
+          the Microsoft.NETCore.App (RID-agnostic/identity package). -->
+      <IncludeAllRuntimePackagesInPlatformManifest
+        Condition="'$(IncludeAllRuntimePackagesInPlatformManifest)' == '' AND
+                   '$(BuildFullPlatformManifest)' == 'true'">true</IncludeAllRuntimePackagesInPlatformManifest>
+    </PropertyGroup>
+
+    <MSBuild Projects="@(ProjectReference)"
```

Comment:
Can you double check that your generated platform manifest includes entries for runtime-specific files?  Look for coreclr.dll, System.Private.CoreLib, libcoreclr, etc.

Reply:
Yep, here are some bits:

```
System.IO.Compression.Native.a|Microsoft.NETCore.App.Ref||0.0.0.0
System.IO.Compression.Native.so|Microsoft.NETCore.App.Ref||0.0.0.0
System.IO.Compression.Native.so.dbg|Microsoft.NETCore.App.Ref||0.0.0.0
...
System.Private.CoreLib.dll|Microsoft.NETCore.App.Ref|4.0.0.0|4.6.27403.0
System.Private.CoreLib.pdb|Microsoft.NETCore.App.Ref||0.0.0.0
...
libcoreclr.so|Microsoft.NETCore.App.Ref||0.0.0.0
libcoreclr.so.dbg|Microsoft.NETCore.App.Ref||0.0.0.0
...
coreclr.dll|Microsoft.NETCore.App.Ref||4.6.27403.0
coreclr.pdb|Microsoft.NETCore.App.Ref||0.0.0.0
```

Same as the normal Microsoft.NETCore.App one but with `.Ref`. The package id/version above is a link to the package, if you didn't see it: https://dotnetfeed.blob.core.windows.net/dotnet-core/flatcontainer/microsoft.netcore.app.ref/3.0.0-preview-27406-5/microsoft.netcore.app.ref.3.0.0-preview-27406-5.nupkg